modules/claims_api/lib/brd/brd.rb

```diff
@@ -54,5 +64,12 @@ def client
         f.adapter Faraday.default_adapter
       end
     end
+
+    def rescue_brd(e, service)
+      detail = e.respond_to?(:original_body) ? e.original_body : e
+      log_outcome_for_claims_api(service, 'error', detail)
```

Comment:
future update: I'd say we may want to elaborate it's the BRD call here in the service for simplicities sake, but that can be a later ticket. Just thinking `intake-sites` may make sense to us, but something like `BRD-intake-sites` could be a good elaboration later and simplify where this is coming from. Granted DataDog does a good job at calling out where in the code base it's coming from. Maybe it's nitpick more than future update.